appengine/standard/storage/appengine-client/main.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START sample]
+"""A sample app that uses GCS client to operate on bucket and file."""
+
+# [START imports]
+import logging
+import os
+
+import cloudstorage as gcs
+from google.appengine.api import app_identity
+
+import webapp2
+
+# [END imports]
+
+# [START retries]
+my_default_retry_params = gcs.RetryParams(
```

Comment:
instead of having a temporary variable, just pass this right into `set_default_retry_params`.